backend/app/src/main/java/com/moh/phlat/backend/controller/ProcessDataController.java

```diff
@@ -390,8 +394,10 @@ public ResponseEntity<ResponseMessage> updateProcessDataById(@RequestBody @Valid
 
 	@PreAuthorize("hasAnyRole(@roleService.getAllRoles())")
 	@GetMapping("/getformfields/header")
-	public String getAllHeader() {
-		return dbUtilityService.getVariablesByTableNameSortedById("PROCESS_DATA");
+	public ResponseEntity<ResponseMessage> getAllHeader() {
+	    List<UiColumnName> list = null;
+		list = tableColumnInfoService.getUiColumnNames("PROCESS_DATA");
```

Comment:
```suggestion
		list = tableColumnInfoService.getDispalyColumnNames("PROCESS_DATA");
```

Reply:
I made the change as per your suggestion.